### ISSUE TITLE
Rewrite &amp; to & (Which break legal xml format)

### ISSUE THREAD

**Issue by alfnielsen** [pmahend1/PrettyXML#4]
When running the format all instance of  `&amp;` is rewritten to `&`
This break the xml file!

**Comment by pmahend1** [comment]
@alfnielsen could you please provide the XML string to investigate?

**Comment by alfnielsen** [comment]
This xml file will change `&amp;` to `&` and `&lt;` to `<`
```xml
<?xml version="1.0" encoding="ISO-8859-1" standalone="yes"?>
<CATALOG><IDENT><SENDER_ID>xxx</SENDER_ID></IDENT><ITEMS><ITEM ACTION="UPDATE"><ITEM_NO>yyy</ITEM_NO><ITEM_NAME>Test &amp; &lt; Test</ITEM_NAME></ITEM></ITEMS></CATALOG>

```

**Comment by pmahend1** [comment]
@alfnielsen It should be fixed in version **0.0.8**
